# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/longrun/makefile-install.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2001-02-14 11:53:22.000000000 +0100
+++ ./Makefile	2003-07-12 13:50:17.000000000 +0200
@@ -1,3 +1,8 @@
+
+PREFIX = /usr
+DESTBIN = $(PREFIX)/sbin
+DESTMAN = $(PREFIX)/man/man1
+
 all: longrun README
 
 longrun: longrun.c
@@ -8,3 +13,14 @@
 
 clean:
 	-rm -f longrun README
+
+install:
+	install -m 755 -d $(DESTBIN)
+	install -m 755 longrun $(DESTBIN)
+	install -m 755 -d $(DESTMAN)
+	install -m 644 longrun.1 $(DESTMAN)
+
+uninstall:
+	rm -f $(DESTBIN)/longrun
+	rm -f $(DESTMAN)/longrun.1
+
